src/main/java/ui/issuepanel/IssuePanel.java

```diff
@@ -36,7 +37,11 @@
         new KeyCodeCombination(KeyCode.N, KeyCombination.CONTROL_DOWN);
     private final KeyCombination defaultSizeWindow =
         new KeyCodeCombination(KeyCode.D, KeyCombination.CONTROL_DOWN);
+
+    // At the moment, apparently, whenever the screen refreshes, all previously lit up comments will
+    // lose the highlight.
```

Comment:
It is an observation, or rather what I believe is the current behaviour of the program. I was actually going to confirm whether this was actually the current behaviour. If yes, then we can make an issue to fix it.